### PR TITLE
Broken link fixed week29

### PR DESCRIPTION
**Pull request by atikmapari**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@atikmapari : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@atikmapari : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@atikmapari : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/tools/kusto-explorer-using.md [hunk 1]
```diff
@@ -167,7 +167,9 @@ You can use parametrized queries with deep-linking.
 1. Provide a parameter for every query parameter in the URI
 In this case:
```
Comment (orspod) [inline data-explorer/kusto/tools/kusto-explorer-using.md:168]:
```suggestion
1. Provide a parameter for every query parameter in the URI, such as:
```

**Comment by PRMerger7** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**